lib/usecases/run/GetAllRunsUseCase.js

```diff
@@ -143,6 +145,28 @@ class GetAllRunsUseCase {
             if (triggerValues) {
                 filteringQueryBuilder.where('triggerValue').oneOf(...triggerValues);
             }
+            if (aliceL3Current) {
+                /**
+                 * The alice l3 current computed from its absolute value and polarity
+                 * @param {function} literal function to create an object representing a database literal expression
+                 * @return {Object} the object representing the column
+                 */
+                const computedColumn = ({ literal }) => literal("alice_l3_current * (1 - 2 * (`alice_l3_polarity` = 'NEGATIVE'))");
```

Comment:
Why is this? This do not seem coherent with the pre-defined versions? Also, `IF` is way more readable

Reply:
applied IF, what are predefined filtering options on fronted has nothing to do with filtering here , alice L3/dipole current are numeric properties so should be filterable as numbers, not as categorical properties